src/test/run-pass/issue-17904.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2014 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
```

Comment:
Currently this file doesn't pretty print correctly: https://github.com/alexcrichton/rust/commit/acef2920d46350cbd9dc4f495072e3cc61bf7545

(it's fine to ignore pretty printing for now)